src/scripts/main.js

```diff
@@ -2,7 +2,19 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tablElement = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  const trElement = document.createElement('tr');
+
```

Comment:
it is a good practice to use object destructing 
`const { name, sex, born, died } = person`